vignettes/manual/SimpleNBox.Rmd

```diff
@@ -12,10 +14,13 @@ Contents:
 Hector's land C sink is divided into three pools: vegetation ($C_v$), detritus ($C_d$), and soil ($C_s$).
 These sinks are global by default, but they can be divided into $n$ "biomes" with unique parameters and initial pool sizes.
 
-The total C flux from atmosphere to land at time $t$ ($F_{L}(t)$) is the net of C sequestration by photosynthesis (net primary productivity, $NPP_i(t)$) and C release by heterotrophic respiration ($RH_i(t)$) summed across each biome $i$, and global land use change ($LUC(t)$):
+The total C flux from atmosphere to land at time $t$ ($F_{L}(t)$; also known as Net Biome Production, NBP) is the net of C sequestration by photosynthesis (net primary productivity, $NPP_i(t)$) and C release by heterotrophic respiration ($RH_i(t)$) summed across each biome $i$, and global land use change ($LUC(t)$):
 
 $$ F_{L}(t) = \sum_{i=1}^{n} \left( NPP_i(t) - RH_i(t) \right) - LUC(t)  $$
 
+(Another significant component of land-atmosphere C exchange is emissions from fire.
```

Comment:
If it were me, I'd leave this out of this section, as a piece of our development roadmap is not really what users are looking for here, nor is this the place where users would generally go to look for information on future development.    We could, however,  just mention that fires are not included, if we think that users might expect it to be.  If we do that, we should mention it in the paragraph above, not as a parenthetical down here.

Reply:
Addressed in 0d4ed96. @bpbond You were the one that originally requested I mention fire -- let me know if it looks acceptable to you.

Reply:
That's a good point–normally NBP would include fire. This looks fine for now, thanks.

Reply:
@bpbond is fire coming into Hector in the form of LUC emissions?

Reply:
No; it's not there at all currently afaik.